.github/workflows/node.js.yml

```diff
@@ -0,0 +1,42 @@
+
+name: Chaos-Master CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
Maybe depending on free minutes, we want to use just 20 or 22? If that even matters for us

Reply:
use only one please, there is no need to run on multiple environments

Reply:
Done